### PR TITLE
Update area14.md

### PR DESCRIPTION
**Pull request by alikeshadow**

### CODE REVIEW

**Review by OriginalRiser — APPROVED** [review]

**Comment by unix-system** [comment]
LGTM

**Comment by Ulferno** [comment]
LGTM

**Comment by Daxrentha** [comment]
LGTM

File: docs/changelogs/area14.md [hunk 1]
```diff
@@ -13,6 +13,19 @@ This web page is managed and updated by the Manufacturing Department.
...
+This update was worked on by Alikeshadow.
+
+### Added
+- SCAR-H to THI Scribe+.
+- The taser for people on the EC team.
+- Senior+ access 173's chamber.
+
+### Fixed
+- Vehicle Depot doors being accessible by Class-D.
+- Coffee tray permissions.
```
Comment (unix-system) [inline docs/changelogs/area14.md:26]: What's the scope on the Coffee tray permissions? I'd like some additional security scoping on this to ensure best practice.

File: docs/changelogs/area14.md [hunk 2]
```diff
@@ -13,6 +13,19 @@ This web page is managed and updated by the Manufacturing Department.
 * [Detailed Weapon Statistics](https://docs.google.com/spreadsheets/d/17A-Qn2cw7yGLl_R0BbsM4ODvLzdZUj_Z9p6ZllokMis/edit?usp=sharing)
 
 
+## 30th August, 2021
+This update was worked on by Alikeshadow.
+
+### Added
+- SCAR-H to THI Scribe+.
+- The taser for people on the EC team.
+- Senior+ access 173's chamber.
```
Comment (Ulferno) [inline docs/changelogs/area14.md:22]: What senior? A CI senior, an SD Senior? I must know which senior!

**Pull request merged**